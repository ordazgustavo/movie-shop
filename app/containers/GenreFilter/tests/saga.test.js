/**
 * Test sagas
 */

/* eslint-disable redux-saga/yield-effects */
// import { take, call, put, select } from 'redux-saga/effects';
// import genreFilterSaga from '../saga';

// const generator = genreFilterSaga();

describe('genreFilterSaga Saga', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false)
  })
})
