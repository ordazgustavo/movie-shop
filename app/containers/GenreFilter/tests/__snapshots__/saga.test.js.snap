// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getGenres Saga should dispatch the getGenresSuccess action if it requests the data successfully 1`] = `
Object {
  "@@redux-saga/IO": true,
  "combinator": false,
  "payload": Object {
    "args": Array [
      "/genre/movie/list?language=en-US&page=1",
    ],
    "context": null,
    "fn": [Function],
  },
  "type": "CALL",
}
`;
